pages/products/[slug].tsx

```diff
@@ -36,12 +42,107 @@ export const getStaticProps = async ({ params }: { params: IParams }) => {
 };
 
 const Page = ({ product }: { product: Product }) => {
+  const [selectedSize, setSelectedSize] = useState<string | undefined>(undefined);
+  const [selectedQuantity, setSelectedQuantity] = useState<number>(0);
   if (!product) return <>Loading...</>;
+
+  const {
+    name,
+    description,
+    colors,
+    price,
+    sys,
+    rating,
+    imagesCollection,
+    department,
+    size,
+  } = product;
   return (
-    <div>
-      <h1>Product</h1>
-      <h2>{product.name}</h2>
-    </div>
+    <Layout
+      seo={{
+        title: `${product.name} | QuackStore`,
+        description: `| QuackStore`,
+        canonicalUrl: `https://quackstore.com/${product.slug}`,
```

Comment:
This is wrong, should be https://quackstore.com/products

Reply:
fixed